website/docs/r/policy_set.html.markdown

```diff
@@ -49,6 +49,27 @@ resource "tfe_policy_set" "test" {
 }
 ```
 
+Local policies (non-VCS based usage, relies on Policy Set Version upload).
```

Comment:
I'd move this above the 'manually-specified policies' one above, keeping related things together.

```suggestion
Manually uploaded policy set, in lieu of VCS:
```